autoload/lsp/omni.vim

```diff
@@ -70,19 +70,22 @@ function! lsp#omni#complete(findstart, base) abort
         endfor
         let l:curpos = getcurpos()
         let l:left = strpart(getline(l:curpos[1]), 0, l:curpos[2]-1)
-        let s:completion['startcol'] = matchstrpos(l:left, l:refresh_pattern)[1]
-        if s:completion['startcol'] == -1
-            let s:completion['startcol'] = strlen(l:left)
+
+        " Initialize the default startcol. It will be updated if the completion items has textEdit.
+        let s:completion['startcol'] = 1 + matchstrpos(l:left, l:refresh_pattern)[1]
```

Comment:
The `startcol` sholud be 1-origin due to `col` name convention. It's just a refactor.

Reply:
can we use utils from here instead/ https://github.com/prabirshrestha/vim-lsp/blob/master/autoload/lsp/utils/position.vim

Reply:
I think current implementation is correct. `matchstrpos` and `startcol` both are byte index. these have just the differences of 1-origin or 0-origin.